Cinema system/Cinema system/Services/CinemaService.cs

```diff
@@ -0,0 +1,84 @@
+﻿using CinemaSystem.Interfaces;
+using CinemaSystem.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CinemaSystem.Services
+{
+    public class CinemaService : ICinemaService
+    {
+
+        private readonly CinemaDBContext _db;
+
+        public CinemaService(CinemaDBContext db)
+        {
+            _db = db;
+        }
+
+        public bool IsCinemaExists(int cinemaId)
+        {
+            return _db.Cinemas.Any(c => (c.Id == cinemaId && !c.IsDeleted));
+        }
+
+        public bool IsCinemaExists(CinemaView cinema)
+        {
+            return _db.Cinemas.Any(c => (c.Title == cinema.Title && c.City == cinema.City && !c.IsDeleted));
+        }
+
+        public int CreateCinema(CinemaView cinema)
+        {
+            var generatedCinema = new Cinema
+            {
+                Title = cinema.Title,
+                City = cinema.City
+            };
+
+            _db.Cinemas.Add(generatedCinema);
+            _db.SaveChanges();
+
+            return generatedCinema.Id;
+        }
+
+        public void EditCinema(int cinemaId, CinemaView cinema)
+        {
+            var editCinema = _db.Cinemas.Find(cinemaId);
+
+            editCinema.Title = cinema.Title;
+            editCinema.City = cinema.City;
+
+            _db.SaveChanges();
+        }
+
+        public IEnumerable<CinemaView> GetCinemas()
+        {
+            return _db.Cinemas.Where(c => c.IsDeleted == false)
+                              .Select(c => new CinemaView
```

Comment:
Use just one level of indent (i.e. one tab) instead of alignment on previous call.
Also it's better to move .Where() on a new line